test/verifier/verifier_test.go

```diff
@@ -179,6 +181,10 @@ func TestVerifier(t *testing.T) {
 				})
 				var ve *ebpf.VerifierError
 				if errors.As(err, &ve) {
+					// Rename object file to avoid subsequent runs to overwrite it,
+					// so we can keep it for CI's artifact upload.
+					os.Rename(objFile, path.Join(*ciliumBasePath, "bpf", fmt.Sprintf("%s_%d.o", bpfProgram.name, i)))
```

Comment:
nit: this could reuse `name` here which is defined here: https://github.com/cilium/cilium/pull/25862/files#diff-941fe2ebb6a483a4cc321ef3fff0f95f576b45f810bce9416eaefb35b46587b7R133

```suggestion
					os.Rename(objFile, path.Join(*ciliumBasePath, "bpf", name + ".o"))
```

Reply:
Makes sense, I'll update. Thanks!